src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -67,8 +80,9 @@ class AmendmentTableImpl : public AmendmentTable
 
     void addInitial () override;
 
-    AmendmentState* addKnown (const char* amendmentID, const char* friendlyName,
-        bool veto) override;
+    AmendmentState* addKnown (uint256 const& amendmentID,
```

Comment:
Agree, will mark class as final. It lives in a cpp file. If someone wants to inherit from it later, it's self contained in the cpp, so it'll be easy to see the consequences of removing the "final".